packages/react-scripts/utils/isReactScriptsLinked.js

```diff
@@ -0,0 +1,7 @@
+var path = require('path');
+var fs = require('fs');
+
+module.exports = () => {
+  var localReactScriptsPath = path.resolve('node_modules/react-scripts');
+  return fs.existsSync(localReactScriptsPath) && fs.lstatSync(localReactScriptsPath).isSymbolicLink();
```

Comment:
@Timer  `fs.lstatSync('unexistingFile')` will throw exception if file does not exist.

Needed for the case `npm run build` at create-react-app root - there is no *react-scripts* at *create-react-app/node_modules*.

Also e2e tests fails if there is no `existsSync` call.